src/layout/ShowRawButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Button, ButtonProps, Code } from '@chakra-ui/react'
+import { useCallback, useState } from 'react'
+import { Trans } from 'react-i18next'
+
+const ShowRawButton = ({ obj, ...props }: { obj: object } & Omit<ButtonProps, 'onClick'>) => {
+  const [showRaw, setShowRaw] = useState(false)
+
+  const toggleRaw = useCallback(() => {
+    setShowRaw(!showRaw)
+  }, [showRaw])
```

Comment:
You can simplify this [by using the `useDisclosure` hook](https://v2.chakra-ui.com/docs/hooks/use-disclosure)